bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -2582,4 +2585,4 @@ R00068,R00068,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,Micr
 R00069,R00069,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUO-00004 MOPAK0000,,20130122,,,
 R00070,R00070,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUO-00005 MOPAK0000,,20130122,,,
 R00106,,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,GA01SUMO-00003-MOPAK,,,,,
-R00297,R00297,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,,,,,,
+R00297,R00297,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,,,,,,
```

Comment:
Not sure I understand this one - isn't the R# and old legacy # and not the real s/n?

Reply:
I didn't actually make a change here. It's just showing a change because this is the new end of the file.

Reply:
Ahhhh gotcha - good.